pkg/resources/external_table.go

```diff
@@ -285,18 +285,22 @@ func UpdateExternalTable(d *schema.ResourceData, meta any) error {
 	if d.HasChange("tag") {
 		unsetTags, setTags := GetTagsDiff(d, "tag")
 
-		err := client.ExternalTables.Alter(ctx, sdk.NewAlterExternalTableRequest(id).WithUnsetTag(unsetTags))
-		if err != nil {
-			return fmt.Errorf("error setting tags on %v, err = %w", d.Id(), err)
+		if len(unsetTags) > 0 {
```

Comment:
this seems unrelated and that it should be in a separate PR?

Reply:
I mean, it's a small change (just enclose within if-s). I can extract it into another pr, but that was just a thing I saw when I was looking for example uses of Tag changes.